Cereal/CerealDecoder.swift

```diff
@@ -42,14 +42,14 @@ public struct CerealDecoder {
     private let reverseRange: StrideThrough<Int>
     private let indexMap: [String: Int]?
 
-    private func itemForKey(key: String) -> CoderTreeValue? {
+    fileprivate func itemForKey(key: String) -> CoderTreeValue? {
```

Comment:
I think this should be `itemFor(key:)`, according to new swift guidelines, will patch it asap